app/components/Texting/components/FormView/FormView.jsx

```diff
@@ -1,20 +1,26 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import {
+  normalize,
+  validatePhoneNumber,
+} from '../../../../utils/validatePhoneNumber';
 import Heading from './Heading';
 import Privacy from './Privacy';
 import Buttons from './Buttons';
 import styles from './Form.module.scss';
 
 const initialState = {
   recipientName: '',
-  phoneNumber: '',
   agreed: false,
 };
 
 const FormView = ({ service, handleSubmit, closeModal }) => {
   const [state, setState] = useState(initialState);
-  const { recipientName, phoneNumber, agreed } = state;
+  const { recipientName, agreed } = state;
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [isValidNumber, setIsValidNumber] = useState(false);
```

Comment:
I don't think you want to create a separate state hook to hold whether the number is valid. State is supposed to be the single source of truth of the state of a component, and but here, whether a phone number is valid is always derived from the number itself. Adding a separate state hook only increases fragility, since it allows for the following invalid states to be represented:

- `phoneNumber` holds a value that is valid, but `isValidNumber` is false
- `phoneNumber` holds a value that is invalid, but `isValidNumber` is true

Since whether the phone number is valid can be fully derived from the number itself, you should just derive it from the number instead of creating a separate store for it.

Could you just get rid of this state hook and just compute `isValidNumber` from `phoneNumber` as a normal variable?